contrib/verifybinaries/README.md

```diff
@@ -6,7 +6,7 @@ Make sure you obtain the proper release signing key and verify the fingerprint w
 
 ```sh
 $ gpg --fingerprint "Bitcoin Core binary release signing key"
-pub   4096R/36C2E964 2015-06-24 [expires: 2017-02-13]
+pub   4096R/36C2E964 2015-06-24 [expires: 2022-02-10]
```

Comment:
```suggestion
pub   4096R/36C2E964 2015-06-24
```

Reply:
Is it ok if the `gpg` example output does not match the real one? I mean that `gpg` actually prints `expires: ` string.

Reply:
Alternatively you can change the text to something like:

`gpg --fingerprint "01EA 5486 DE18 A882 D4C2  6845 90C8 019E 36C2 E964"` should return a key and not an error.

Reply:
I don't see a problem with matching the real `gpg` output.

Reply:
or replace it with YYYY-MM-DD, so it's clear there needs to be something there, but this document doesn't need to be updated every time for it. There's already too many things that need to be updated after an expiration date bump.

Reply:
`gpg --fingerprint "Bitcoin Core binary release signing key" | cut -f1 -d'['` :-)

Reply:
The only reason I added the gpg output here, is to have the fingerprint written down in our doc. The reason was not to have some gpg output here that is supposed to be constantly updated whenever a date or version changes. I think we have better things to do and solve.